app/ai/voice/agents/breeze_buddy/handlers/transport/utils/response_transform.py

```
@@ -245,6 +245,184 @@ def scale_by_exponent(value: Any, args: Dict[str, Any]) -> Any:
+    except (TypeError, ValueError):
+        return value
+    if args.get("skip_zero") and number == 0:
+        return value
+    divide_by = args.get("divide_by")
+    if isinstance(divide_by, (int, float)) and divide_by:
+        number = number / divide_by
+    decimals = args.get("decimals", 0)
+    if not isinstance(decimals, int) or decimals < 0:
+        decimals = 0
+    if decimals == 0:
+        body = str(int(round(number)))
```

Ruff (0.16.2)

[warning] Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)